frontend/src/routes/Identities/Users/UserDetails.tsx

```diff
@@ -0,0 +1,73 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { PageSection, Stack, Text } from '@patternfly/react-core'
+import { DescriptionList, DescriptionListTerm, DescriptionListGroup, DescriptionListDescription } from '@patternfly/react-core'
+import { useParams } from 'react-router-dom-v5-compat'
+import { useTranslation } from '../../../lib/acm-i18next'
+import { useQuery } from '../../../lib/useQuery'
+import { listUsers } from '../../../resources/rbac'
+import { useMemo } from 'react'
+import AcmTimestamp from '../../../lib/AcmTimestamp'
+import { AcmLoadingPage } from '../../../ui-components'
+
+const UserDetails = () => {
+  const { t } = useTranslation()
+  const { id = undefined } = useParams()
+  const { data: users, loading } = useQuery(listUsers)
+
+  const user = useMemo(() => {
+    if (!users || !id) return undefined
+    return users.find((u) => u.metadata.uid === id || u.metadata.name === id)
+  }, [users, id])
+
+  if (loading) {
+    return (
+      <PageSection>
+        <AcmLoadingPage />
+      </PageSection>
+    )
+  }
+
+  if (!user) {
+    return (
+      <PageSection>
+        <div>{t('User not found')}</div>
+      </PageSection>
+    )
+  }
+
+  return (
+    <PageSection>
+      <PageSection variant={'light'}>
+        <Text style={{ fontSize: '1.25rem', fontFamily: 'RedHatDisplay' }} component={'h2'}>
+          {t('General information')}
+        </Text>
+        <Stack hasGutter>
+          <DescriptionList isHorizontal={false}>
+            <DescriptionListGroup>
+              <DescriptionListTerm>{t('Full name')}</DescriptionListTerm>
+              <DescriptionListDescription>{user.fullName ?? '-'}</DescriptionListDescription>
```

Comment:
it is not consistent with the `Unknown user` label from title
<img width="1078" height="520" alt="image" src="https://github.com/user-attachments/assets/36e9f9a1-4ec2-4713-bb41-996e982fff71" />
my question is... is the `fullName` really present at user level? I can't see it on openshift schema https://docs.okd.io/3.9/rest_api/apis-user.openshift.io/v1.User.html, am I missing anything here?